cloudflare/resource_cloudflare_device_posture_rule.go

```diff
@@ -59,32 +59,64 @@ func resourceCloudflareDevicePostureRule() *schema.Resource {
 			},
 			"input": {
 				Type:     schema.TypeList,
-				Required: true,
+				Optional: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"id": {
-							Type:     schema.TypeString,
-							Optional: true,
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "serial number rule: the id of the serial number list",
```

Comment:
Ideally, these descriptions should match the website documentation you've updated. In the future, the website documentation will be generated from this field. You can just include a description of the field itself (no need for things like `file rule:`, etc)

Reply:
I'll update these.